### PR TITLE
[PHP 8.0] add `str_contains` function

### PR DESCRIPTION
**Pull request by IonBazan**
Waiting for https://wiki.php.net/rfc/str_contains RFC.

### CODE REVIEW

File: src/Php80/Php80.php [hunk 1]
```diff
@@ -26,6 +26,11 @@ public static function fdiv($dividend, $divisor)
         return (float) @($dividend / $divisor);
     }
 
+    public static function str_contains($haystack, $needle)
```
Comment (nicolas-grekas) [inline src/Php80/Php80.php]: please add all type declarations

Comment (IonBazan) [reply src/Php80/Php80.php]: Are you sure? We are not using them in other functions yet.

**Comment by IonBazan** [comment]
Rebased 🚀

File: src/Php80/Php80.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Php80/Php80.php]: See #234

File: src/Php80/Php80.php [hunk 2]
```diff
@@ -49,6 +49,11 @@ public static function preg_last_error_msg()
         }
     }
 
+    public static function str_contains($haystack, $needle)
+    {
+        return '' === $needle || false !== \strpos($haystack, $needle);
```
Comment (nicolas-grekas) [inline src/Php80/Php80.php]: no need for the `\`

File: src/Php80/Php80.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Php80/Php80.php]: (please rebase again while doing so)

File: tests/Php80/Php80Test.php [hunk 3]
```diff
@@ -87,6 +87,24 @@ public function testPregMalformedUtf8Offset()
...
+    /**
+     * @covers \Symfony\Polyfill\Php80\Php80::str_contains
+     */
+    public function testStrContains()
+    {
+        $this->assertTrue(str_contains('abc', ''));
+        $this->assertTrue(str_contains('abc', 'a'));
+        $this->assertTrue(str_contains('abc', 'bc'));
+        $this->assertTrue(str_contains('abc', 'abc'));
+        $this->assertTrue(str_contains('abc', 'abc'));
```
Comment (noniagriconomie) [inline tests/Php80/Php80Test.php]: Twice same test here

File: src/Php80/Php80.php (hunk 1 above)
Comment (IonBazan) [reply src/Php80/Php80.php]: OK, typehints added, changes rebased again 😉

**Comment by nicolas-grekas** [comment]
Thank you @IonBazan.

**Pull request merged**